frontend/app/components/TaskDescription.server.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import { FetchError } from 'ofetch'
+
+const props = defineProps<{
+  taskId: string | string[] | undefined
+}>()
+
+const description = ref('')
+const toast = useToast()
+
+if (props.taskId === undefined) {
+  showError({
+    statusCode: 404,
+    message: 'Nie znaleziono zadania',
+  })
+} else {
+  try {
+    const task_id = String(props.taskId)
+    const address = '/tasks/description/{task_id}'
+    const { data: response } = await useApi(address, {
+      path: { task_id },
+    })
+
+    if (!response.value) {
+      showError({
+        statusCode: 404,
+        message: 'Zadanie nie zostało znalezione',
```

Comment:
Yeah 2 errors with different wording just to confuse users hihi. Create function for showing this error